File: testprojects/scala_main/pipelines/DONOT_OPEN_SCALA/code/src/main/scala/org/main/scla_dep_mgmt/graph/OrderBy_3.scala
package org.main.scla_dep_mgmt.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt.config.ConfigStore._
import org.main.scla_dep_mgmt.config.Context
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object OrderBy_3 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.orderBy(col("c1").asc,
               col("c2").asc,
               col("c3").asc,
               col("c4").asc,
               col("c5").asc,
               col("c6").asc,
               col("c7").asc,
               col("c8").asc,
               col("c9_udf1").asc,
               col("c9_udf2").asc
    )

}

File: testprojects/master_python_mainone/pipelines/PYTHON_DEP_MANAGEMENT_ALL/code/job/graph/OrderBy_3.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def OrderBy_3(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.orderBy(
        col("customer_id").asc(), 
        col("first_name").asc(), 
        col("last_name").asc(), 
        col("phone").asc(), 
        col("email").asc(), 
        col("country_code").asc(), 
        col("account_open_date").asc(), 
        col("account_flags").asc()
    )
